docs/dev/versioning.md

```diff
@@ -17,8 +17,10 @@ At a `point` release, there should be no changes beyond bug fixes.
 
 Valid version numbers are described in [PEP 440](https://peps.python.org/pep-0440/).
 
-[Release candidates](https://peps.python.org/pep-0440/#pre-releases) should have versions like `1.7.0rc1`, `1.7.0rc2`.
-[Development versions](https://peps.python.org/pep-0440/#developmental-releases) should look like `1.8.0.dev0`, with a commit hash optionally appended as a local version identifier (e.g. `1.8.0.dev2+g00ad77b`).
+[Pre-releases](https://peps.python.org/pep-0440/#pre-releases)
+:   should have versions like `1.7.0a1`, `1.7.0b1`, or `1.7.0rc2`.
```

Comment:
```suggestion
[Pre-releases](https://peps.python.org/pep-0440/#pre-releases)
:   should have versions like `1.7.0rc1` or `1.7.0rc2`.
```

I'm not sure why this file is changing. AFAIK, we don't have plans to change anything about versioning or doing alphas/ betas.

Reply:
Ah sure, it’s just confusing to mention versions like that then